src/adaptors/mod.rs

```diff
@@ -507,69 +507,6 @@ where
     }
 }
 
-/// An iterator adaptor that steps a number elements in the base iterator
-/// for each iteration.
-///
-/// The iterator steps by yielding the next element from the base iterator,
-/// then skipping forward *n-1* elements.
-///
-/// See [`.step()`](crate::Itertools::step) for more information.
-#[deprecated(note = "Use std .step_by() instead", since = "0.8.0")]
-#[allow(deprecated)]
-#[derive(Clone, Debug)]
-#[must_use = "iterator adaptors are lazy and do nothing unless consumed"]
-pub struct Step<I> {
-    iter: Fuse<I>,
```

Comment:
`StepBy` does not fuse afaik. I'm fine with that.

Reply:
I did not know that.
I don't mind dropping that commit. @jswrenn's call then.

Reply:
I'm also fine with removing this despite the small difference in behavior.